designer/client/index.tsx

```diff
@@ -1,5 +1,4 @@
 import React from "react";
-import ReactDOM from "react-dom";
```

Comment:
Removing this is giving the following ts error down in `ReactDOM.render(...`
`'ReactDOM' refers to a UMD global, but the current file is a module. Consider adding an import instead.`